_data/simple-icons.json

```
@@ -1809,6 +1809,12 @@
             "hex": "064F8C",
             "source": "https://www.kitware.com/platforms/"
         },
+        {
+            "title": "CNCF",
+            "hex": "231F20",
```

```suggestion
            "hex": "2A0552",
```
more prominent website color